cluster/gce/gci/configure-helper.sh

```diff
@@ -520,10 +520,21 @@ function start-etcd-servers {
     rm -f /etc/init.d/etcd
   fi
   prepare-log-file /var/log/etcd.log
-  prepare-etcd-manifest "" "4001" "2380" "200m" "etcd.manifest"
+  prepare-etcd-manifest "" "4001" "2380" "150m" "etcd.manifest"
 
   prepare-log-file /var/log/etcd-events.log
-  prepare-etcd-manifest "-events" "4002" "2381" "100m" "etcd-events.manifest"
+  prepare-etcd-manifest "-events" "4002" "2381" "50m" "etcd-events.manifest"
+}
+
+# Starts Calico policy controller pod.
+# Specifically, replaces the variable values in the manifest, and copies it to
+# /etc/kubernetes/manifests.
+function start-calico-policy-controller {
+  local -r temp_file="/tmp/calico-policy-controller.manifest"
+  cp "${KUBE_HOME}/kube-manifests/kubernetes/gci-trusty/calico-policy-controller.manifest" "${temp_file}"
+  PUBLIC_IP=$(ifconfig eth0 | grep 'inet ' | awk '{print $2}')
```

Comment:
when I run this locally I get "addr:172.17.128.66" but when I run it on GCI I get a bare IP.  This feels fragile.  how about `hostname -i` ?  That is fragile in a different way...